### ISSUE TITLE
Rough sketch of front-end

### ISSUE THREAD

**Issue by sheel12** [sheel12/PhysicsBasicsApp#1]
Discuss the overall layout of the app from the user's perspective.

**Comment by sheel12** [comment]
We need to discuss the following screens:
login
summary
Welcome screen
new card
update card
enter transaction

**Comment by dchopra001** [comment]
Yes we should plan what the user's perspective should look like.
